### PR TITLE
Fix #1880, static page for 'shield_study_1'.

### LINKED ISSUE #1880: Static page for Firefox shield
We need to get a static page up for Firefox shield. This will be similar to pages we've done for various other projects in the past, and is essentially a way to get around the "this add-on isn't on AMO" warning in the install flow.

@MattGrimes is going to supply the HTML for the page, but we just need to make sure its routed correctly in our URLs so it shows up at addons.mozilla.org.

Hoping to push this out next week. @EnTeQuAk can you assist on this please?

### PR DESCRIPTION
**Pull request by EnTeQuAk**
Based on https://github.com/mozilla/addons-server/pull/1940

Fixes #1880

### CODE REVIEW

**Comment by gregglind** [comment]
(I have some more copy changes, should I open a new PR?)

**Comment by EnTeQuAk** [comment]
I just rebased the branch and pulled in your changes.

File: src/olympia/pages/templates/pages/shield_study_1.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+<ol>
+<li>Type <code>about:addons</code> into the location bar, and press enter.</li>
+<li>Find <code>x-screen-draw-performance-shield-study-1</code> in the addons list.</li>
+<li>Click the <code>remove</code> button.</li>
+<li>If you opt out of the study early we will ask you to fill out a survey. We’re interested in hearing about your experience, even if you didn’t participate in the entire study.</li>
+</ol>
+<p>Opting out of a study does not prevent you from participating in future studies.</p>
+<h2 id="data-we-will-collect">Data we will collect</h2>
+<ul>
+<li>General Firefox Usage Data, in accordand with the <a href="https://www.mozilla.org/en-US/privacy/">Firefox Privacy Policy</a></li>
```
Comment (gregglind) [inline src/olympia/pages/templates/pages/shield_study_1.html]: typo.  'accordance'

**Comment by EnTeQuAk** [comment]
Alrighty, this is now the final revision and ready for review.

**Comment by diox** [comment]
English only ??

File: src/olympia/pages/urls.py [hunk 2]
```diff
@@ -47,4 +47,10 @@
         TemplateView.as_view(template_name='pages/sunbird.html'),
         name='pages.sunbird'),
     url('^sunbird/', lambda r: perma_redirect(reverse('pages.sunbird'))),
+
+    url('^shield-study-1$',
+        TemplateView.as_view(template_name='pages/shield_study_1.html'),
+        name='pages.shield_study_1'),
+    url('^shield-study-1/',
+        lambda r: perma_redirect(reverse('pages.sunbird'))),
```
Comment (diox) [inline src/olympia/pages/urls.py]: Copy/paste fail :)

I'm not sure you need that. Couldn't you just include a trailing slash in the true URL instead ?

Comment (EnTeQuAk) [inline src/olympia/pages/urls.py]: dang, yeah, fixed and removed the non-slash version.

**Comment by gregglind** [comment]
diox, english only.  Addon and and page is public.  Recruitment funnel should filter non-EN.

**Comment by diox** [comment]
r+

**Pull request merged**